scripts/add-cluster.sh

```diff
@@ -369,7 +392,12 @@ CLUSTER_LABEL=""
 if [[ ${JOINING_CLUSTER_TYPE_NAME} == "member" ]]; then
     CLUSTER_LABEL="cluster-role.toolchain.dev.openshift.com/tenant: ''"
 fi
-OWNER_CLUSTER_NAME=$(echo "${CLUSTER_JOIN_TO_TYPE_NAME}-${CLUSTER_JOIN_TO_NAME}${MULTI_MEMBER}" | head -c 63)
+
+# see comment for TOOLCHAINCLUSTER_NAME variable that explains owner cluster name composition.
+CLUSTERNAME_FIXED_PART="${CLUSTER_JOIN_TO_TYPE_NAME}-${MULTI_MEMBER}"
+CLUSTERNAME_LENGTH_TO_REMOVE="${#CLUSTERNAME_FIXED_PART}"
+CLUSTERNAME_LENGTH_TO_KEEP=$((62-CLUSTERNAME_LENGTH_TO_REMOVE))
+OWNER_CLUSTER_NAME="${CLUSTER_JOIN_TO_TYPE_NAME}-${CLUSTER_JOIN_TO_NAME:0:CLUSTERNAME_LENGTH_TO_KEEP}${MULTI_MEMBER}"
```

Comment:
I'm afraid that this will result in a different owner cluster name if the `MULTI_MEMBER == 1` - the inner part created by truncating `CLUSTER_JOIN_TO_NAME` will have one character less than the actual name because if the `MULTI_MEMBER` is set to 1, then you calculate the `CLUSTERNAME_FIXED_PART` already with the additional char, but if the script is executed again, but with different parameters, then you still don't know if the `MULTI_MEMBER` will be set or not.

Reply:
@MatousJobanek as discussed, I'm marking the `ownerClusterName` field as `obsolete` for now, and dropped the creation logic see : [0fa93d7](https://github.com/codeready-toolchain/toolchain-cicd/pull/83/commits/0fa93d7ef8b829347bbc3172b31aace2754d3d96)

I've triggered new e2e test job in the paired PR as well.

The removal of the field is postponed for when we will revisit the logic and structure of the `toolchaincluster` CRD.